test/unittests/eof_validation_test.cpp

```diff
@@ -1211,3 +1211,198 @@ TEST_F(eof_validation, max_nested_containers)
 
     add_test_case(code, EOFValidationError::success);
 }
+
+// Summary of validity of combinations of referencing instructions with instructions inside
+// referenced containers.
+// Rows are instructions referencing subcontainers or rules for top-level container.
+// Columns are instructions inside referenced subcontainer or none if it ends with REVERT/INVALID.
```

Comment:
regardless of the topic below, I would be more strict on the terminology here: "top-level" or "ends" aren't well defined. An alternative is to remove "top-level" altogether and split the row into `account.code` and `InitcodeTransaction` or sth.
```suggestion
// Rows denote the way subcontainers containing the validated code are referenced by parent containers. The extra row for the top-level container means either one set for an account or found in the `initcodes` field of an `InitcodeTransaction`.
// Columns are terminating instructions used inside the validated code of such a container or none if code only terminates with REVERT/INVALID.
```

Reply:
I rephrased and restructured the table a bit.